packages/flutter_tts/example/tizen/Runner.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Tizen.NET.Sdk/1.1.5">
+﻿<Project Sdk="Tizen.NET.Sdk/1.1.6">
```

Comment:
Just a question, is this change necessary?

Reply:
There weren't any problems that I know of, I just updated it to a latest compatible version.